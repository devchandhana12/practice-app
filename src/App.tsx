import React from "react";
import "./App.css";
import Login from "./pages/Login";
import "bootstrap/dist/css/bootstrap.min.css";
import { BrowserRouter, Routes, Route } from "react-router-dom";
import SignUp from "./pages/Signup";
import Home from "./pages/Home";
// contexts
import AuthProvider from "./contexts/AuthContext";
function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <AuthProvider>
          <Routes>
            <Route path="/home" element={<Home />} />
            <Route path="/Login" element={<Login />} />
            <Route path="/Signup" element={<SignUp />} />
          </Routes>
        </AuthProvider>
      </BrowserRouter>
    </div>
  );
}

export default App;
