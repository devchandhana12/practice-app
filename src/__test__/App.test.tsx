import React from "react";
import { render, cleanup } from "@testing-library/react";
import App from "../App";
import "resize-observer-polyfill";

global.ResizeObserver = jest.fn(() => ({
  observe: jest.fn(),
  unobserve: jest.fn(),
  disconnect: jest.fn(),
}));

afterEach(cleanup);
it("should take a snapshot", () => {
  const { asFragment } = render(<App />);

  expect(asFragment()).toMatchSnapshot();
});
