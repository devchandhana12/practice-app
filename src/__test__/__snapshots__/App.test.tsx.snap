// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div
        class="recharts-responsive-container"
        style="background-color: black; margin-top: 50px; width: 70%; height: 500px; min-width: 0;"
      />
    </div>
  </div>
</DocumentFragment>
`;
